Resources/Prototypes/Recipes/Crafting/Graphs/improvised/rolling_pin.yml

```
@@ -0,0 +1,15 @@
+- type: constructionGraph
+  id: WoodenRollingPin
+  start: start
+  graph:
+    - node: start
+      edges:
+        - to: rollingpin
+          steps:
+            - material: WoodPlank
+              amount: 2
+            - material: MetalRod
+              amount: 1
+              doAfter: 2
+    - node: rollingpin
+      entity: RollingPin
```

```suggestion
  - node: start
    edges:
    - to: rollingpin
      steps:
      - material: WoodPlank
        amount: 2
      - material: MetalRod
        amount: 1
        doAfter: 2
  - node: rollingpin
    entity: RollingPin
```
Fix the indentation, otherwise this looks good.